subsys/net/openthread/rpc/server/ot_rpc_if_server.c

```diff
@@ -199,3 +200,23 @@ static void ot_rpc_cmd_if_send(const struct nrf_rpc_group *group, struct nrf_rpc
 
 NRF_RPC_CBOR_CMD_DECODER(ot_group, ot_rpc_cmd_if_send, OT_RPC_CMD_IF_SEND, ot_rpc_cmd_if_send,
 			 NULL);
+
+static void ot_get_extaddr(const struct nrf_rpc_group *group, struct nrf_rpc_cbor_ctx *ctx,
```

Comment:
Consider if this should be added to `ot_rpc_ctrl` instead (we will probably have to add `otLinkGetExtendedAddress` serialization anyway).

Reply:
done.